README.md

```diff
@@ -115,8 +115,43 @@ board.print_connections()  # Print updated connections
 # Disconnect all channels
 board.disconnect_all_channels()
 ```
+### Example 3: Switching from Simulated to I2C-Controller
 
-### Example 3: Resetting the Board
+This example showcases Example 2, but showcases using the I2CDriverBoardController
+instead of the simulated controller.
+
+```python
+from aliaroaccessoryboards import AccessoryBoard, BoardConfig, I2CDriverBoardController
+import i2cdriver
+
+# Create a configuration for the board
+board_config = BoardConfig.from_device_name('32ch_instrumentation_switch')
+
+# Initialize i2cdriver instance to connected USB port
+driver = i2cdriver.I2CDriver("COM5")
+
+# Pick a starting address for your I2C bus to target
+target_i2c_address = 0
```

Comment:
We should use something like `8` for the example I2C address, which I would expect to be the first valid value.